test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -32,7 +32,7 @@ export default class BaseFormObject {
   }
 
   fieldByLabelExists(label, options = undefined) {
-    return within(this.el).getByLabelText(new RegExp(label), options).exists;
```

Comment:
fieldByLabelExists() will fail for cases where label doesn't exist, if we use `getByLabelText()`
Using `queryByLabelText()` allows us to use this method for non existent label scenarios

Reply:
Ohh yes 🤕 

I'll leave it up to you if you feel its worth adding that function. 🙏